verification/src/compiler/fetcher.rs

```diff
@@ -7,3 +7,7 @@ pub trait Fetcher {
     type Error: Send + Sync + 'static;
     async fn fetch(&self, ver: &CompilerVersion) -> Result<PathBuf, Self::Error>;
 }
+
+pub trait VersionList {
+    fn all_versions(&self) -> Vec<&CompilerVersion>;
```

Comment:
As I remember @sevenzing suggested to return `BTreeSet<_>` in some place. What if we return `BTreeSet` here creating a contract between the trait and its users that the method would return versions in a sorted order?

@leviska What do you think?

Reply:
I suggested use BTreeMap for type of `releases` in `CompilerFetcher` so that  versions are always sorted

But I don't think we need to return BTreeSet and introduce some contract in trait. don't see enough reason to do it